src/js/background/context-menu.js

```diff
@@ -165,6 +172,11 @@ const relayContextMenus = {
       }
     }
 
+    const phoneMask = relayContextMenus.utils.getPhoneMask();
```

Comment:
I found the issue! This is an async function.

```
await relayContextMenus.utils.getPhoneMask()
```

~Note: per my previous review, this is returning true with a free Relay account. Noting that here.~

Reply:
Wow! Not just identifying an issue I missed, but also debugging it to find the root cause and provide a fix - thank you so much! Fixed in b19c445

(Now if only [we had some tools](https://mozilla-hub.atlassian.net/browse/MPP-2512) that would've allowed me to catch this without relying on a very thorough human review :sweat_smile: )

Reply:
_If only! 🥲_